app/collins/controllers/actions/ipaddress/GetPoolsAction.scala

```diff
@@ -38,14 +39,25 @@ case class GetPoolsAction(
           } else {
             IpAddresses.getPoolsInUse()
           }
-        format(pools)
+        val ipmiPools: Set[String] =
+          if (all) {
+            IpmiInfo.AddressConfig.map(_.poolNames).getOrElse(Set())
+          } else {
+            // TODO: Change this to only return active pools?
+            IpmiInfo.AddressConfig.map(_.poolNames).getOrElse(Set())
```

Comment:
This is the one thing I don't like about this currently. How IpAddress pools are handing this is just querying the DB and returning all unique fields. Since this is not in the DB at all for IPMI pools we would have to run a very expensive query to figure out what all the active pools are. The other option is writing a migration to go with an evolution which I believe is the correct path forward.